packages/fluentui/docs/src/prototypes/table/ResponsiveTable.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { Table } from '@fluentui/react';
+import calculateBreakpoints, { Column, Breakpoint } from '../../utils/calculateBreakpoints';
+import { TableProps } from '../../../../react/src/components/Table/Table';
+
+interface ResponsiveTableProps extends TableProps {
+  label: string;
+  header: {
+    items: Column[];
+  };
+}
+
+const responsiveTableID = 'ResponsiveTable';
+
+const ResponsiveTable = (props: ResponsiveTableProps) => {
+  const {
+    rows,
+    label,
+    header: { items }
+  } = props;
+
+  const columns = items.map(({ title }) => title);
+
+  const [breakpoints, setBreakpoints] = React.useState<Breakpoint[]>();
+
+  // Generate the stylesheet once and for all
+  React.useLayoutEffect(() => {
+    const breakpoints = Array.from(calculateBreakpoints(items, { _: 16 }));
```

Comment:
Is the `_` default value? If yes, can we give it more descriptive name? It took me some time until I realized what it is (if I got it correctly :))